apps/sdk-e2e/src/integration/transaction-kinetic-sdk-e2e.spec.ts

```diff
@@ -54,40 +54,43 @@ describe('KineticSdk (e2e)', () => {
     expect(source).toBe(aliceKeypair.publicKey)
   }, 60000)
 
-  it('should throw an error when there are less than 1 transactions in a batch', async () => {
-    try {
-      const destinations: Destination[] = []
-      await sdk.makeTransferBatch({ destinations, owner: aliceKeypair })
-    } catch (error) {
-      expect(error.toString()).toContain('Error: At least 1 destination required')
-    }
+  it('should throw an error when there are less than 1 transaction in a batch', async () => {
+    const destinations: Destination[] = []
+
+    await expect(async () => await sdk.makeTransferBatch({ destinations, owner: aliceKeypair })).rejects.toThrow(
+      'At least 1 destination required',
+    )
   })
 
   it('should throw an error when there are more than 15 transactions in a batch', async () => {
-    try {
-      const destinations: Destination[] = []
-      const destination = { destination: bobKeypair.publicKey, amount: '15' }
-      for (let i = 0; i <= 15; i++) {
-        destinations.push(destination)
-      }
-      await sdk.makeTransferBatch({ destinations, owner: aliceKeypair })
-    } catch (error) {
-      expect(error.toString()).toContain('Error: Maximum number of destinations exceeded')
+    const destinations: Destination[] = []
+    const destination = { destination: bobKeypair.publicKey, amount: '15' }
+    for (let i = 0; i <= 15; i++) {
+      destinations.push(destination)
     }
+
+    await expect(async () => await sdk.makeTransferBatch({ destinations, owner: aliceKeypair })).rejects.toThrow(
+      'Maximum number of destinations exceeded',
+    )
   })
 
   it('should fail when one account does not exist in batch transfer', async () => {
-    try {
-      const destinations: Destination[] = []
-      destinations.push({ destination: bobKeypair.publicKey, amount: '15' })
-      const kp = Keypair.random()
-      destinations.push({ destination: kp.publicKey, amount: '12' })
-      await sdk.makeTransferBatch({ destinations, owner: aliceKeypair })
-    } catch (error) {
-      const errorData = error.response.data.error
```

Comment:
This `catch` block never ran. The `makeTransferBatch` promise does not reject (maybe it should?) when this happens, but instead it resolves with a `Transaction` object containing the error.